models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml

```diff
@@ -0,0 +1,23 @@
+models:
+  - name: machine-translation-nar-de-en-0001
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: narnmt
+          vocabulary_file: tokenizer_tgt/vocab.json
+          merges_file: tokenizer_tgt/merges.txt
+          output_name: preds
+        inputs:
+          - name: "tokens"
+            value: 'tokens'
+            type: INPUT
+
+    datasets:
+      - name: WMT_de_en
+        postprocessing:
```

Comment:
Please add preprocessing

Reply:
There are preprocessing steps in dataset definitions. Resolved

Reply:
done